packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -657,9 +657,11 @@ class _LineItemState extends State<LineItem> {
   DebuggerController _debuggerController;
 
   String _previousHoverWord = '';
+  bool _hasMouseExited = false;
 
   void _onHoverExit() {
     _showTimer?.cancel();
+    _hasMouseExited = true;
     _removeTimer = Timer(LineItem._hoverDelay, () {
```

Comment:
this should be a separate delay. I don't think this delay should be made longer.

Reply:
I disagree. We are less likely to "flash" a hover card if a user accidentally moves outside the line.

Reply:
I agree with jacob. Look at the difference between the hover card show and hover card hide delays in IntelliJ. There is a longer delay to show than exit.

![hover-intellij](https://user-images.githubusercontent.com/43759233/119862680-99a17300-becd-11eb-8c47-3fefcc0ff23b.gif)

Reply:
I can shorten the delay but we don't want it too small as we want to allow users to hover "into" the card which keeps it around. Otherwise it's frustrating to expand variables displayed in the hover card.

Reply:
i would hope this code path isn't needed to make the case of hovering into a card work. ideally the area treated as in the card and the current location should have a border so you don't need to rely on the delay to move over the hover crd.

Reply:
That is not the case. Note that the card is rendered below the source line which is why this is required. This also mirrors the Chrome DevTools behavior. There appears to be a grace period where you can move your mouse outside the hovercard and into before it is removed.